### DISCUSSION TITLE
Disabling keyboard backlight when using battery

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#2245] [Ideas]
Hello. First of all, thank you very much for developing Ghelper.

I wish there was an option to automatically turn off the keyboard backlight when using battery.

thank you !

**Comment by seerge** [comment]
@cheonmux hello,

It’s already possible. App remembers backlight brightness separately for plugged in / on battery states. And would set last used brightness automatically

**Reply by cheonmux** [reply]
I've now tested each brightness and it works really well!

thank you! :)
